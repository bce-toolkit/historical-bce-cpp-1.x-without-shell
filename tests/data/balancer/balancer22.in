Al+HNO3=Al(NO3)3+NO+H2O