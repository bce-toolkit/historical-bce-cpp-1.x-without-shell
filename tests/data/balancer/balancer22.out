Al+4HNO3=Al(NO3)3+NO+2H2O